service-commands/src/setup.js

```diff
@@ -614,18 +619,21 @@ const allUp = async ({
 
   const setup = [
     [Service.NETWORK, SetupCommand.UP],
-    [Service.SOLANA_VALIDATOR, SetupCommand.UP],
-    [Service.SOLANA_VALIDATOR, SetupCommand.HEALTH_CHECK]
+    [Service.SOLANA_VALIDATOR_PREDEPLOYED, SetupCommand.UP],
+    [Service.SOLANA_VALIDATOR_PREDEPLOYED, SetupCommand.HEALTH_CHECK]
```

Comment:
do we need SOLANA_VALIDATOR_PREDEPLOYED if buildSolana == true ?

Reply:
We still need a validator running, the `buildSolana` flag will re-build and deploy the programs & reset the configs
The flag should only be used by the eng-platform team while developing the solana programs

Reply:
oh got it, this was my bad, this makes sense, the validator will always be the pre-packaged one but the actual programs can be overwritten with the locally